voxblox_ros/src/tsdf_server.cc

```diff
@@ -157,11 +179,11 @@ void TsdfServer::insertPointcloud(
     // Convert the PCL pointcloud into our awesome format.
     // TODO(helenol): improve...
     // Horrible hack fix to fix color parsing colors in PCL.
-    for (size_t d = 0; d < pointcloud_msg->fields.size(); ++d) {
+    /*for (size_t d = 0; d < pointcloud_msg->fields.size(); ++d) {
```

Comment:
Does this need to be removed? This was making many pointclouds not work for me without.